frontend/src/utils/game_utils.js

```diff
@@ -82,6 +84,7 @@ export const getSteps = (path, gameMap, speed) => {
             x = round(x);
             y = round(y);
         }
+        [x, y] = [xf, yf];
```

Comment:
Doesn't seem like you use these or `z` later on but might still be good to have as a reminder for any future maintenance.